runtime/queries/purescript/highlights.scm

```diff
@@ -117,8 +117,9 @@
 
  (row_field (field_name) @variable.other.member)
  (record_field (field_name) @variable.other.member)
- (record_accessor (variable) @variable.other.member)
- (exp_record_access (variable) @variable.other.member)
+ (record_field (field_pun) @field)
```

Comment:
```suggestion
 (record_field (field_pun) @variable.other.member)
```

nvim's `@field` is `@variable.other.member` in helix capture names

Reply:
Oh, this `@field` one is a typo, that's my bad sorry. Just pushed a change. Otherwise this file was always Helix-based. Sorry I don't understand what's your thoughts on the rest of it? `#match?` has always been there and Helix' Haskell queries (where these come from) have it as well.

Reply:
So from what I understand only the `@field` typo would need a change.

Reply:
Updated to the latest commit